OpenRA.Utility/Program.cs

```diff
@@ -62,7 +62,8 @@ static void Main(string[] args)
 			{
 				var command = (IUtilityCommand)Activator.CreateInstance(commandType);
 				var kvp = new KeyValuePair<Action<ModData, string[]>, Func<string[], bool>>(command.Run, command.ValidateArguments);
-				actions.Add(command.Name, kvp);
+				if (!actions.ContainsKey(command.Name))
+					actions.Add(command.Name, kvp);
```

Comment:
I added a stub `mono --debug OpenRA.Utility.exe $mod --map-import` that will do auto-detection and re-run the utility with the best guess. This way we retain compatibility and robustness.